test/powershell/Language/Scripting/DeserializedMethods.Tests.ps1

```diff
@@ -0,0 +1,40 @@
+﻿Describe "DeserializedMethods" -Tags "CI" {
+
+    BeforeAll {
+        try {
+            $temp = [io.path]::GetTempFileName()
+        } catch {
+            Remove-Item $temp -force -ea silentlycontinue
+            return
+        }
+    }
+
+    AfterAll {
+	        Remove-Item $temp -force -ea silentlycontinue
+        }    
```

Comment:
Fixed.

---
In reply to: [76680218](https://github.com/Powershell/Powershell/pull/2118#discussion_r76680218) [](ancestors = 76680218)